lib/model/card/card.dart

```diff
@@ -1,12 +1,22 @@
+import 'package:treasure_of_the_high_seas/model/card/quest_lines.dart';
+import 'package:treasure_of_the_high_seas/model/card/special_card_types.dart';
+
 import '../game_state.dart';
 import 'action/card_action.dart';
 import 'card_types.dart';
 
 abstract class Card {
   final String name;
   final CardType type;
+  final int questStage;
+  final QuestLine questLine;
+  final SpecialType specialType;
```

Comment:
:thinking:  It feels a bit messy to have these on a base class when they only apply to quest cards. I guess up until now we've gotten away with the simple `CardType` prop because the types haven't diverged beyond this. Now they are diverging, perhaps it's time for separate classes?

Reply:
In particular, if we had a `QuestCard` we could make these properties non-optional, which is a bit simpler to think about (and gets rid of the risk of us forgetting to implement one or more props for future quest cards)